packages/lyra/tests/lyra.test.ts

```diff
@@ -195,6 +195,51 @@ describe("lyra", () => {
     expect(searchResult.hits[0].id).toBe(id2);
   });
 
+  it("Shouldn't returns deleted documents", async () => {
+    const db = new Lyra({
+      schema: {
+        txt: "string",
+      },
+      stemming: false,
+    });
+
+    await db.insert({ txt: "stelle" });
+    await db.insert({ txt: "stellle" });
+    await db.insert({ txt: "scelte" });
+
+    const search = await db.search({ term: "stelle" });
+
+    const id = search.hits[0].id;
+
+    await db.delete(id);
+
+    const serach2 = await db.search({ term: "stelle" });
+
+    expect(serach2.count).toBe(1);
```

Comment:
This assertion should check that the search hits do not include the deleted ID, not that the count is 1.